PIITokenizer.ts

```diff
@@ -0,0 +1,47 @@
+
+
+import { DateSanitizer, GenderSanitizer, StringSanitizer } from "./Sanitizer";
+import { QGramExpander, DateExpander } from "./Expander";
+import { PrivacyPreservingTokenizer } from "./PrivacyPreservingTokenizer";
+
+export class PIITokenizer {
+  stringSanitizer: StringSanitizer;
+  dateSanitizer: DateSanitizer;
+  genderSanitizer: GenderSanitizer;
+
+  dateExpander: DateExpander;
+  qGramExpander: QGramExpander;
+
+  tokenizer: PrivacyPreservingTokenizer;
+
+  constructor(bloomFilterLength: number = 200, numberOfHashFunctions: number = 20, privacyBudget: number = 3.0) {
```

Comment:
I don't remember why the paper used 200. We should use a longer length for starters so we don't have to revise this as often as the dataset gets large. Any downsides to starting with a 500?